carbon-design-system/carbon-addons-iot-react
# fix(side-nav): adjust z-index on overlay to cover table

Closes #2363 

**Summary**

- updates the z-index of the active overlay to cover the table

**Change List (commits, features, bugs, etc)**

- add z-index to active overlay

**Acceptance Test (how to verify the PR)**

- on the sidenav story
- shrink the window
- open the sidenav
- overlay should appear on top of the table header/pagination

**Regression Test (how to make sure this PR doesn't break old functionality)**

- make sure the active overlay is blocking anything it shouldn't be in sidenav/suiteheader stories.



**Things to look for during review**
